senders/formatter_test.go

```diff
@@ -192,6 +194,32 @@ func TestSpanLine(t *testing.T) {
 	assert.Equal(t, expected, line)
 }
 
+func TestSpanLineErrors(t *testing.T) {
+	uuid := "00000000-0000-0000-0000-000000000000"
+
+	_, err := SpanLine("", 0, 0, "", uuid, uuid, nil, nil, nil, nil, "")
+	require.Error(t, err)
+	assert.Equal(t, "span name cannot be empty", err.Error())
+
+	_, err = SpanLine("a_name", 0, 0, "00-00", "", uuid, nil, nil, nil, nil, "")
+	require.Error(t, err)
+	assert.Equal(t, "traceId is not in UUID format: span=a_name", err.Error())
+
+	_, err = SpanLine("a_name", 0, 0, "00-00", uuid, "", nil, nil, nil, nil, "")
+	require.Error(t, err)
+	assert.Equal(t, "spanId is not in UUID format: span=a_name", err.Error())
+
+	_, err = SpanLine("a_name", 0, 0, "a_source", uuid, uuid, nil, nil,
+		[]SpanTag{{Key: "", Value: ""}}, nil, "")
```

Comment:
Nit: Struct fields in GO get their zero values if left uninitialized. So you don't have to explicitly set Value to the empty string here.  ```[]SpanTag{{Key: ""}}``` is enough since you are testing what happens if Key is empty.

Reply:
Hm, I'm not sure how I feel about that. I think that "future-me" will better understand the intent of what's being tested by explicitly setting `Value` to empty string.